Source/WebCore/Modules/webxr/WebXRFrame.h

```diff
@@ -85,7 +86,11 @@ class WebXRFrame : public RefCounted<WebXRFrame> {
 
     static TransformationMatrix matrixFromPose(const PlatformXR::FrameData::Pose&);
 
-    struct PopulatedPose;
+    struct PopulatedPose {
+        TransformationMatrix transform;
+        bool emulatedPosition { false };
+    };
+
```

Comment:
Or these

Reply:
This is necessary to fix the build with unified builds.